src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml

```diff
@@ -161,16 +161,20 @@
     - formplayer_deploy
   when: not _should_update_formplayer_in_place
 
-- name: Formplayer monit config
-  become: yes
-  template:
-    src: "monit.formplayer.j2"
-    dest: "/etc/monit/conf.d/formplayer"
-    group: root
-    owner: root
-    mode: 0640
-  notify: reload monit
-  tags: monit
+- block:
+
+  - name: Formplayer monit config
+    become: yes
+    template:
+      src: "monit.formplayer.j2"
+      dest: "/etc/monit/conf.d/formplayer"
+      group: root
+      owner: root
+      mode: 0640
+    notify: reload monit
+    tags: monit
+
+  when: use_monit_for_formplayer == true
```

Comment:
I might be missing the point, but it looks to me like you could forego the `- block` and indentation, and just add the `when: ...` straight to the task.

Reply:
Updated 👍  thanks